File: DatabaseConnector.py
import mysql.connector

class DatabaseConnector:
    _cnx = mysql.connector.connect(host='172.17.0.2', user='root', password='123')
    _cursor = _cnx.cursor()
    _cursor.execute("use food;")

    def get(key):
        DatabaseConnector._cursor.execute('select * from environmental_cost where food_type="{}";'.format(key))
        rows = DatabaseConnector._cursor.fetchall()
        return float(rows[0][1])

File: databaseConnector.js
var mysql = require('mysql');

var con = mysql.createConnection({
  host: "172.17.0.3",
  user: "root",
  password: "123",
  database: "food",
  insecureAuth: true,
});

function get(key) {
  con.connect(function(err) {
    if (err) throw err;
    con.query('select * from environmental_cost where food_type="' + key + '";', function (err, result, fields) {
      if (err) throw err;
      return result[0].kg_co2_per_kg;
    });
  });
}

var result = get("Peas");
console.log("result: " + result);